File: python/example_refactoring.py
# Before refactoring:
# def get_user_info(user):
#     return f'{user.name}, {user.age} years old'

# After refactoring:

def format_user_info(user):
    return f'{user.name}, {user.age} years old'

# Usage:
# print(format_user_info(some_user))

File: csharp/ExampleRefactoring.cs
// Before refactoring:
// public string GetUserInfo(User user) { return user.Name + ", " + user.Age + " years old"; }

// After refactoring:

public string FormatUserInfo(User user) {
    return user.Name + ", " + user.Age + " years old";
}

// Usage:
// Console.WriteLine(FormatUserInfo(someUser));
